.travis.yml

```diff
@@ -6,3 +6,7 @@ python:
 script:
   - make check
   - make unittests
+
+branches:
+  only:
+    - master
```

Comment:
This change will affect autospec forks, right? I generally like to see Travis results when I'm working on a topic branch in a fork...

Reply:
ah, that's a good point. This was just so I didn't get emails for everyone's travis builds in this repo. I'll have to rethink this :(